backend/app/modules/common/string_utils.py

```diff
@@ -0,0 +1,6 @@
+def parse_cids_to_epochs_dict(cidsStr: str) -> dict[int, str]:
```

Comment:
It can be simplified with dict comprahension. Also, please use snake_case when naming env vars in python
```python
def parse_cids_to_epochs_dict(cids_str: str) -> dict[int, str]:
    return {index: element.strip() for index, element in enumerate(cids_str.split(","), start=1)}
```